tests/cli/test_cli_regression_mol.py

```diff
@@ -431,7 +431,7 @@ def test_hyperopt_quick(monkeypatch, data_path, tmp_path):
         "--raytune-search-algorithm",
         "hyperopt",
         "--features-generators",
-        "morgan_count",
+        "morgan_binary",
```

Comment:
Are we no longer testing `morgan_count`?

Reply:
Before we tested `morgan_count` multiple times. `morgan_count` is still tested in `test_optuna_quick`.

Reply:
As long as we have good unit tests for each molecular featurizer and can ensure that the training and hpopt work with `--features-generators`, I think it is fine. I can understand it would be great to cover all possible cases to avoid unexpected errors in the future. For example, we should include a CLI test with data points whose RDKit features might be NaN so we can test that the data point can successfully convert NaN to 0. The drawback is that if we consider every possible case, it would likely take too much time for the checks.